src/js/Core/Storage/SyncedStorage.js

```diff
@@ -138,16 +138,6 @@ SyncedStorage.defaults = Object.freeze({
     "tag_waitlist": false,
     "tag_short": false,
 
-    "hide_owned": false,
-    "hide_ignored": false,
-    "hide_dlcunownedgames": false,
```

Comment:
Removed "hide_dlcunownedgames" as well because I don't the feature carried over from ES. Looking at the implementation I'm not sure how this even works...I can't find a way to associate a DLC to its parent app on search pages.

Reply:
One way is to cache [appdetails](https://store.steampowered.com/api/appdetails?appids=440) for all owned games, to if the parent app for a DLC is owned. I think for most people this a good solution.

Reply:
Isn't it cached for some feature like support info already? And it's only done when the user visits a store page, right? Doing it for all owned games at once is a lot of requests...
Anyway, I think the feature people want is to be able to see a list of all DLC for owned games.

Reply:
Well that feature can be achieved by doing the same thing. Yes you may need a lot of request for large libraries, but you can bulk up to 100 (I believe) appids in a 1 single request, so that should help a lot.  And yes, for support info we also query the same endpoint and cache it, but I think we use filters to only show support info. We need the `basic` filter included to see `dlc` array return.

Reply:
👍 Though we should move this discussion to #952 and focus on implementing that instead of piling on options to hide stuff.